kernel/src/sync/mutex.rs

```diff
@@ -99,7 +102,7 @@ impl<T: ?Sized, S: MutexSupport> Mutex<T, S> {
             let mut try_count = 0;
             // Wait until the lock looks unlocked before retrying
             while self.lock.load(Ordering::Relaxed) {
-                self.support.cpu_relax();
+                unsafe { &*self.support.as_ptr() }.cpu_relax();
```

Comment:
get_mut or get_ref?

Reply:
It's still an unstable feature, and is completely equivalent to deref and take ref again,